### DISCUSSION TITLE
Error when calling the gym_compat module

### DISCUSSION THREAD

**Discussion by rajarshiroychowdhury** [rte-france/Grid2Op#307] [Q&A] [answered]
Seems like the gym_compat module here is throwing some error! Any help is appreciated. 

![image](https://user-images.githubusercontent.com/88808425/164951754-182d1cd7-348f-47ec-a793-584aaa5fb7b4.png)

![error](https://user-images.githubusercontent.com/88808425/164951727-c025777f-551e-4420-96cf-e4ca2603070e.png)

**Comment by BDonnot** [comment, marked as answer]
Hello,

I just looked at the code and this import does not exist anymore.I would recommend to upgrade to the most recent grid2op version (1.6.5)

Let me know if it fixes this issue

**Reply by rajarshiroychowdhury** [reply]
Thanks! Upgraded to the latest version and that solved it!
